search/django/weedid/utils.py

```diff
@@ -51,22 +51,16 @@ def store_tmp_image_from_zip(upload_image_zip, image_dir, full_images):
 
 
 def store_tmp_weedcoco(weedcoco, upload_dir):
-    fs = OverwriteStorage()
```

Comment:
Could you please clarify why use this implementation rather than OverwriteStorage? Not that I think OverwriteStorage is perfect..

Reply:
In the original, `weedcoco` is a file object from the request. In this version, it's JSON which has been put through `fix_compatibility_quirks`.

I took the implementation from `store_tmp_voc_coco` which was also taking JSON (and other parts of the code which are writing out files) - I then replaced `store_tmp_voc_coco` with the new function as they were identical.